vimrc

```diff
@@ -57,16 +67,11 @@ augroup vimrcEx
 
   " Allow stylesheets to autocomplete hyphenated words
   autocmd FileType css,scss,sass setlocal iskeyword+=-
-augroup END
 
-" Softtabs, 2 spaces
-set tabstop=2
-set shiftwidth=2
-set shiftround
-set expandtab
-
-" Display extra whitespace
-set list listchars=tab:»·,trail:·,nbsp:·
+  let g:go_fmt_command = "goimports"
+  autocmd FileType go compiler go
+  autocmd FileType go setlocal listchars=tab:\ \ ,trail:·,nbsp:·
```

Comment:
I believe `vim-go` sets the `filetype` command for us. Strictly speaking, the trailing character highlighting might not be useful as it's stripped out on write, but I kept it in for parity with other filetypes.

Basically, this way we still get the trailing character highlighting:

![](http://caleb.click/doPl/download/Image%202015-11-13%20at%209.47.44%20AM.png)

Whereas with `nolist` I see:

![](http://caleb.click/doZZ/download/Image%202015-11-13%20at%209.48.22%20AM.png)